### PR TITLE
Interfaces of Components

### PR DESCRIPTION
**Pull request by adnan1naeem**
This pull request is of Interfaces of Components in typescript.Interfaces  are made on the basis of react native platform.

### CODE REVIEW

File: Components/TouchableHighlight.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import color from './color';
+import Touchablewithoutfeedback from './TouchableWithoutFeedback';
+
+export interface  ITouchableHight{
```
Comment (amnajunaid) [inline Components/TouchableHighlight.ts]: Interface name is wring please correct it.

File: Components/Drawer.ts [hunk 2]
```diff
@@ -0,0 +1,88 @@
+import color from './color';
+ import  { DrawerLayoutAndroid }  from 'react-native';
+
+export   interface IDrawer{
+
+    /**
+     * The navigation view that will be rendered to the side of the screen and can be pulled in.
+     */
+    renderNavigationView?: 'default'|'black';
```
Comment (amnajunaid) [inline Components/Drawer.ts]: From where default and black are coming? I see no reference.

Comment (adnan1naeem) [reply Components/Drawer.ts]: I have fixed the Interface

File: Components/Drawer.ts [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+     * Function called whenever the navigation view has been closed.
+     */
+
+    onDrawerClose?:()=>void
+
+    /**
+     * Specifies the side of the screen from which the drawer will slide in.
+     */
+
+    drawerPosition?:object
```
Comment (amnajunaid) [inline Components/Drawer.ts]: make it string with 'left' | 'right'

**Pull request merged**